yarn.lock

```diff
@@ -0,0 +1,1294 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
oh no, that got generated when i installed the needed dependencies with yarn, thought that is already in the ``.gitignore``, removing it asap.